pages/osx/open.md

```diff
@@ -6,9 +6,13 @@
 
 `open {{file.ext}}`
 
-- Run a graphical MacOSX application:
+- Run a graphical macOS application:
 
-`open /Applications/{{Application.app}}`
+`open -a {{Application}}`
+
+- Run a graphical macOS app based on the bundle identifier:
```

Comment:
Hrm. In this case I'd say that it's a useful command to have if the default macOS terminal doesn't autocomplete the application id, since you'd still need to go and look up what that other command was if the two go hand-in-hand.

I do understand though that we want to avoid implying that one command is part of the same package / program as another though.